### ISSUE TITLE
Pictures taken using TakePicture has wrong rotation.

### ISSUE THREAD

**Issue by Parixit2411** [yasirkula/UnityNativeCamera#90]
**Description of the bug**

I am using TakePicture to click pictures. Once the pic is clicked it has the wrong rotation. ImageProperties orientation shows unknown. 

**Reproduction steps**

Simply use the TakePicture function with callback parameter and maxSize parameter.

**Platform specs**

Please provide the following info if this is a Unity 3D repository.

- Unity version: 2020.3.29f1
- Platform: Android, not tested on any other platform
- Device: Nokia 6.1 plus OS 10
- How did you download the plugin: Asset Store

**Comment by yasirkula** [comment]
Does NativeCamera.LoadImageAtPath also return a Texture with wrong orientation?

**Comment by Parixit2411** [comment]
I have checked, this problem is only on os 10. On other device it is
working fine.

**Comment by Parixit2411** [comment]
Yes, LoadImageAtPath is also returning texture with the wrong orientation. I have tested it on one other device and it is working fine on it. So I think it might be on Android OS 10 or my particular device.

**Comment by yasirkula** [comment]
I'd like to believe that it's a device-specific issue. If it were an Android 10 issue, I think other similar issues would've arrived by now. Could you replace _NativeCamera.aar_ with the one inside [this zip archive](https://github.com/yasirkula/UnityNativeCamera/files/8240100/NativeCamera.zip) and see if it resolves the issue on the problematic device?

**Comment by Parixit2411** [comment]
Alright, I will use that and check.

**Comment by ernonemacher** [comment]
Facing the same issue, replacing the NativeCamera.aar didn't change anything. `ImageProperties` orientation returns `Rotate180`.

I am testing in a Redmi Note 8, Android 10.

**Comment by Parixit2411** [comment]
As far as I have tested, it happens on some devices with Android 10. So as mentioned earlier, it might be a device issue. Maybe you should try to test it on another device.

**Comment by yasirkula** [comment]
@ernonemacher If you save the image to Gallery with [NativeGallery](https://github.com/yasirkula/UnityNativeGallery), is it displayed correctly there? If it does, does loading the image from gallery with NativeGallery.GetImageFromGallery and NativeGallery.LoadImageAtPath return a correctly oriented Texture?

**Comment by ernonemacher** [comment]
Thanks @yasirkula for your support, it worked. I was using `File.ReadAllBytes`, changing to `NativeCamera.LoadImageAtPath` worked. It wasn't necessary to save the image in the gallery.

**Comment by Parixit2411** [comment]
Yes

**Comment by yasirkula** [comment]
@Parixit2411 Can you replace NativeCamera.aar with the one inside [this zip archive](https://github.com/yasirkula/UnityNativeCamera/files/9791655/NativeCamera.zip) and paste the logcat logs that start with "===" after taking a picture with the camera?
